src/cmd/manage_nsfs.js

```diff
@@ -795,6 +795,12 @@ async function validate_account_args(data, action) {
 
         if (is_undefined(data.access_keys[0].secret_key)) throw_cli_error(ManageCLIError.MissingAccountSecretKeyFlag);
         if (is_undefined(data.access_keys[0].access_key)) throw_cli_error(ManageCLIError.MissingAccountAccessKeyFlag);
+        if (data.nsfs_account_config.gid && (is_undefined(data.nsfs_account_config.uid))) {
+            throw_cli_error(ManageCLIError.InvalidAccountNSFSConfigUID, data.nsfs_account_config);
```

Comment:
```suggestion
            throw_cli_error(ManageCLIError.MissingAccountNSFSConfigUID, data.nsfs_account_config);
```
we already have ManageCLIError.InvalidAccountUID and it's not the appropriate message, I think that missing is the correct term for this issue, please change here and for gid also

Reply:
I will change it, thanks.